pkg/scheduler/framework/session_plugins.go

```diff
@@ -153,7 +153,7 @@ func (ssn *Session) AddJobStarvingFns(name string, fn api.ValidateFn) {
 // Reclaimable invoke reclaimable function of the plugins
 func (ssn *Session) Reclaimable(reclaimer *api.TaskInfo, reclaimees []*api.TaskInfo) []*api.TaskInfo {
 	var victims []*api.TaskInfo
-	var init bool
+	var init = true
```

Comment:
My question is: what does the init variable represent? Why is it better to change it to true?

Reply:
This variable should represent whether it is initialized or not. I think the default value should be false. However, can the variable name be changed? It may cause misunderstanding.

Reply:
I was thinking in that direction. But for me init asks if we need to initialize the list of victims/if it's a first loop. So we can also name it first_iteration, for example. 
It's like thinking -  we want to check if it's not initialized, or we want to check if we need to do init assigment.